kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -160,13 +163,32 @@ abstract class TFrontendService(name: String)
     val ipAddress = getIpAddress
     val configuration =
       Option(req.getConfiguration).map(_.asScala.toMap).getOrElse(Map.empty[String, String])
-    val sessionHandle = be.openSession(
-      protocol,
-      userName,
-      req.getPassword,
-      ipAddress,
-      configuration)
-    sessionHandle
+
+    val sessionHandleGuid =
+      configuration.get(KyuubiReservedKeys.KYUUBI_SESSION_HANDLE_GUID)
+    val sessionHandleSecret =
+      configuration.get(KyuubiReservedKeys.KYUUBI_SESSION_HANDLE_SECRET)
```

Comment:
in which case, we need these?

Reply:
recreate the protocol and connect to the old session

Reply:
in which case we need the original session?

Reply:
the original session means original context, If we create a new session, the context is loss